test/test_utils.go

```diff
@@ -72,10 +73,45 @@ func GetNormalPodOwnerRefList() []metav1.OwnerReference {
 	return ownerRefList
 }
 
+// BuildTestReplicaController creates a test replication controller
+func BuildTestReplicaController(name string, replicas int) *v1.ReplicationController {
+	rc := &v1.ReplicationController{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "default",
+			Name:      name,
+		},
+		Status: v1.ReplicationControllerStatus{
+			Replicas: int32(replicas),
+		},
+	}
+	return rc
+}
+
+// BuildTestReplicaSet creates a test replica set
+func BuildTestReplicaSet(name string, replicas int) *apps.ReplicaSet {
+	rs := &apps.ReplicaSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "default",
+			Name:      name,
+		},
+		Status: apps.ReplicaSetStatus{
+			Replicas: int32(replicas),
+		},
+	}
+	return rs
+}
+
 // GetReplicaSetOwnerRefList returns the ownerRef needed for replicaset pod.
-func GetReplicaSetOwnerRefList() []metav1.OwnerReference {
+func GetReplicaSetOwnerRefList(rsName string) []metav1.OwnerReference {
+	ownerRefList := make([]metav1.OwnerReference, 0)
```

Comment:
This change passes the name of replicaset so that i can re-use this method to create ownerrefs of different RS